telegram/_poll.py

```diff
@@ -86,16 +87,26 @@ class PollAnswer(TelegramObject):
 
     Args:
         poll_id (:obj:`str`): Unique poll identifier.
-        user (:class:`telegram.User`): The user, who changed the answer to the poll.
-        option_ids (Sequence[:obj:`int`]): 0-based identifiers of answer options, chosen by the
-            user. May be empty if the user retracted their vote.
+        voter_chat (:class:`telegram.Chat`): The chat that changed the answer to the poll,
+            if the voter is anonymous.
+
+            .. versionadded:: NEXT.VERSION
+        user (:class:`telegram.User`): The user that changed the answer to the poll,
```

Comment:
maybe add note on backwards compatibility? samew below on the attribute.
both `user` and `voter_chat` are optional.
please also update the docstring of `option_ids`